### DISCUSSION TITLE
more compact view

### DISCUSSION THREAD

**Discussion by ttes0** [pawelsalawa/sqlitestudio#4626] [Q&A]
I'm using ver 321 but would like to have a more compact view in 333 (as I have in 321). Is this possible? I'm on debian 10/buster
![Screenshot_2022-12-19_13-55-03](https://user-images.githubusercontent.com/120829806/208537300-89def3aa-cae4-4f61-a23f-07066d6bce03.png)

**Comment by pawelsalawa** [comment]
If I see correctly, it's all about the fonts, isn't it?

How about version 3.4.1? How does that look like for you?

**Reply by pawelsalawa** [reply]
Also the 3.4.1 has font configuration back (it was missing in 3.3.x).

**Comment by ttes0** [comment]
341 doesn't run for me. I get error -- perhaps debian 10 libraries too old?

![Screenshot_2022-12-19_16-50-40](https://user-images.githubusercontent.com/120829806/208556679-2d569913-e2e9-4b32-8764-fd9e4b685107.png)

**Reply by pawelsalawa** [reply]
Oh, you're using Ubuntu 20.04 or older. The 3.4.1 version is built on Ubuntu 22.04.

Nevertheless, you still can change font size in 3.3.x using the very window that you've shown - the CSS console. Try adding there:
```
* {
    font-size: 9px;
}
```
and hit Apply. See how that works for you.

**Comment by ttes0** [comment]
Thank you, that did the trick with the tab and menu options fonts. Is there a way to decrease the bottom padding in table/view data display (too much space below text) and font size, and font size of left sided database display?

![Screenshot_2022-12-19_13-41-08](https://user-images.githubusercontent.com/120829806/208732393-a0abbde8-5000-402f-b1f0-3a2affdd24ed.png)
![Screenshot_2022-12-19_13-40-32](https://user-images.githubusercontent.com/120829806/208732417-69e1f86c-57da-4aa6-9418-1c9b4a086881.png)

**Reply by pawelsalawa** [reply]
Frankly speaking I don't know. I don't know the "Qt CSS" well enough. Sorry.
In the end, you could upgrade your Ubuntu to 22.04 and use SQLiteStudio 3.4.1, where you can adjust font sizes and data views are adjusted to font sizes accordingly.

**Comment by ttes0** [comment]
I must apologize for wasting your time. It seems that using a combination of your suggestion of  "*{font-size: 12px;}" and QTableView::item{ border: 0px;  padding: 2px;} and the configuration dialogue in sqlitebrowser, I now have the desired results. Thank you so much though for your help. Also, my debian (not ubuntu) install still has a few good years left in it before upgrade. I don't need the latest software for my purposes.
![Screenshot_2022-12-20_14-43-36](https://user-images.githubusercontent.com/120829806/208781690-f9da7a30-8c8a-4bde-a838-66cbf409aa2d.png)

![Screenshot_2022-12-20_14-50-05](https://user-images.githubusercontent.com/120829806/208782201-20f99cd6-bb24-4639-9ad9-5af9eef5a7ff.png)
